content/en/docs/overview.md

```diff
@@ -96,18 +96,18 @@ To facilitate better communications across the skill sets, respecting everyone's
 We followed the design patterns of other large, complex projects like Antoni Gaudí's design of [The Sagrada Familia](https://simple.wikipedia.org/wiki/Sagrada_Fam%C3%ADlia).
 The [sketch, prototype, build approach](https://stevelasker.blog/2020/07/31/sketch-prototype-build/) would enable the various experts to focus on their component, while understanding where they plug-into other components of the design.
 
-As a result, we identified the following stages of the Notary effort:
+As a result, we identified the following stages of the Notary Project effort:
 
 1. Define Requirements
 1. Build Prototypes
 1. Validate prototypes - learning, refining requirements, iterating prototypes
 1. Author a Notary Spec
 
-See the [Notary project on GitHub](https://github.com/notaryproject/notaryproject/tree/main/status-updates) for updates to the stages of development and areas of focus. 
+See [the Notary Project on GitHub](https://github.com/notaryproject/notaryproject/tree/main/status-updates) for updates to the stages of development and areas of focus. 
```

Comment:
suggestion. either we update content on this link, or we point users back to our Github milestones/roadmap.

Reply:
```suggestion
See milestones of respective Notary Project sub-projects for updates to development and areas of focus. 
```

Reply:
fixed